test/specs/behaviors/testDefinitions.ts

```diff
@@ -124,20 +124,68 @@ definitions.push({
   },
 })
 
-// Example: Adds attribute 'aria-disabled=true' to 'trigger' component's part based on the property 'disabled'.
+function testMethodConditionallyAddAttribute(
+  parameters,
+  attributeToBeAdded,
```

Comment:
would suggest to rearrange these method's args so its intent will be easier to read. The following option might work better:

```js
function testMethodConditionallyAddAttribute(
  parameters,
  component,
  propertyDependsOn,
  attributeToBeAdded,
  valueOfAttributeToBeAddedIfTrue,
  valueOfAttributeToBeAddedOtherwise
)
```

Reply:
done, bonus parameter added